modules/reviewing-pod-status.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * support/investigating-pod-issues.adoc
+
+[id="reviewing-pod-status_{context}"]
+= Reviewing Pod status
+
+Query Pod status and error states. Query a Pod's associated deployment configuration and review base image availability.
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`.
+* `skopeo` is installed.
```

Comment:
Be consistent in starting your prereqs the same way. You have both "Install..." and "... is installed."

Review all prereqs throughout PR for this.

Reply:
I have now resolved this throughout the PR.